test/tasks/Regression.t.sol

```diff
@@ -331,6 +332,29 @@ contract RegressionTest is Test {
         _assertDataToSignNestedMultisig(multisigTask, actions, expectedDataToSign);
     }
 
+    /// @notice Expected call data and data to sign generated by manually running the DisableModule template at block 8098229 on sepolia
```

Comment:
If you don't have a .env file on the task it probably gets the latest block on Sepolia, I guess here 8098229 it's incorrect even if it doesn't matter because it's a comment. It wouldn't be bad to set a block on the .env and edit it here so the tests are always consistent.

Reply:
OK, good call.